0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -167,14 +115,36 @@ Content-Type: application/spsp4+json
 }
 ```
 
-### Payment Setup
+## Model of Operation
+
+### Establishing a connection
+
+We assume that the SPSP Client knows the SPSP Server's SPSP Endpoint (see [Payment pointers](../0026-payment-pointers/0026-payment-pointers.md)).
+
+1. The user's SPSP Client queries the SPSP Server's SPSP Endpoint.
+
+2. The SPSP Endpoint responds with the SPSP Server's info, namely the ILP address (`destination_account`) and the shared secret (`shared_secret`) to be used in STREAM. 
+    * The `destination_account` MUST be used as the STREAM `destinationAccount`.
+    * The `shared_secret` MUST be decoded from base64 and used as the STREAM `sharedSecret`.
+  
+    It MAY respond with additional information if it is an invoice server or a pull payment server. For more information, see [SPSP Invoices](../0035-spsp-invoices/0035-spsp-invoices.md) and [SPSP Pull Payments](../0036-spsp-pull-payments/0036-spsp-pull-payments.md).
+
+3. The SPSP Client establishes a STREAM connection to the SPSP Server using the SPSP Server's ILP address and shared secret.
+
+### Simple push payment
+
+Given the open STREAM connection, the SPSP Client begins sending ILP packets of value.
+  1. The SPSP Client will adjust their STREAM `sendMax` to reflect the amount they're willing to send.
+  2. The SPSP Server will adjust their STREAM `receiveMax` to reflect the amount they're willing to receive.
+  3. The SPSP Client's and Server's STREAM Modules will move as much value as possible while staying inside these bounds.
+  4. If the SPSP Client reaches their `sendMax`, they end the stream and the connection. If the SPSP Server reaches their `receiveMax`, they will end the stream and the connection.
+
+### Simple data transmission
```

Comment:
I see your point. I can also remove this whole section if everybody agrees @sharafian @adrianhopebailie @justmoon.

Reply:
if we specify the encoding as UTF-8 it doesn't add much. But if you say JSON should be sent (we could say you send one json object per stream) then it would allow multiple extensions of SPSP to work in the same connection and a server could ignore the fields it doesn't understand (just like the SPSP response itself)

Reply:
I'd drop everything after line 134 or put them in a section called "Examples" or similar.
This is just repeating what is already said in the STREAM spec.

Reply:
Moved it to _Examples_.